src/ansys/geometry/core/modeler.py

```diff
@@ -166,6 +166,21 @@ def create_design(self, name: str) -> "Design":
             )
         return self._designs[design.design_id]
 
+    def get_active_design(self) -> "Design":
+        """
+        Get the active design on the modeler object.
+
+        Returns
+        -------
+        Design
+            Design object already existing on the modeler.
+        """
+        for key, _ in self._designs.items():
+            if self._designs[key]._is_active:
+                return self._designs[key]
```

Comment:
```suggestion
        for _ , design in self._designs.items():
            if design ._is_active:
                return design
```

Reply:
No need to just extract the keys and then extract the value given a key... you can loop on the values directly.